unity-renderer/Assets/DCLServices/EmotesCatalog/EmotesCatalogBridge.cs

```diff
@@ -17,6 +17,7 @@ private class AddEmotesResponse
     }
 
     public event Action<WearableItem[]> OnEmotesReceived;
+    public event Action<string, string> OnEmoteRejected;
```

Comment:
I think it would be better to have an exception here instead of `string, string`. Or if not... a delegate where you can name each string.

I had to go to the invokation to understand what's the first string and what's the second.